app/signin/page.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+import React, { useState } from "react";
+import styles from "../../styles/SigninPage.module.css";
+
+import { loginSuccess } from "../../utils/commonUtil";
+import { postReqeustApi } from "../../utils/requestApi";
+import SignInfoInputComponent from "../../components/SignInfoInputComponent";
+
+// email 정규식
+const regex = new RegExp("[a-z0-9]+@[a-z]+.[a-z]{2,3}");
+
+const SigninPage = () => {
+  const [signinInfo, setSigninInfo] = useState({
+    email: false,
+    password: false,
+  });
+
+  const [emailErrorText, setEmailErrorText] = useState("");
+  const [pwErrorText, setPwErrorText] = useState("");
+
+  const validCheck = (name: string, value: string) => {
+    let isValid = false;
+    if (name === "email") {
+      if (value === "") {
+        // 오류
+        setEmailErrorText("이메일을 입력해 주세요.");
+      } else if (!regex.test(value)) {
+        // 오류
+        setEmailErrorText("올바른 이메일 주소가 아닙니다.");
+      } else {
+        // 정상
+        setEmailErrorText("");
+        isValid = true;
+      }
+    } else if (name === "password") {
+      if (value === "") {
+        setPwErrorText("비밀번호를 입력해 주세요.");
+      } else {
+        setPwErrorText("");
+        isValid = true;
+      }
+    }
+    return isValid;
+  };
+
+  const changeSigninInfo = (e: any) => {
+    const isValid = validCheck(e.target.name, e.target.value);
+
+    setSigninInfo({
+      ...signinInfo,
+      [e.target.name]: isValid,
+    });
+  };
+
+  const signinFormSubmit = (e: any) => {
+    e.preventDefault();
+    if (signinInfo.email && signinInfo.password) {
```

Comment:
함수 내부에서도 하는 일이 다른것들끼리는 개행으로 구분지어주면 코드를 읽는데 더 좋아요~
```suggestion
    e.preventDefault();
    
    if (signinInfo.email && signinInfo.password) {
```